.github/workflows/deploy.yaml

```diff
@@ -2,8 +2,8 @@ name: HOPR Deploy
 on:
   push:
     branches: ['master', 'release/**', 'avado']
-    paths-ignore:
-      - 'docs/**'
+    paths:
```

Comment:
Why did you change this?

Reply:
The Jobs were running on Whitepaper changes